### PR TITLE
Add helper to check if department is covered for weather alert bulletin

### PR DESCRIPTION
**Pull request by oncleben31**
This modification is needed to build a clean correction for https://github.com/home-assistant/core/issues/38635

### CODE REVIEW

File: src/meteofrance/helpers.py [hunk 1]
```diff
@@ -39,13 +40,24 @@ def get_phenomenon_name_from_indice(int_phenomenon: int, lang: str = "fr") -> st
 
 
 def is_coastal_department(department_number: str) -> bool:
-    """Iidentify when a second bulletin is availabe for coastal risks (Helper)."""
+    """Identify when a second bulletin is availabe for coastal risks (Helper)."""
     result = False
     if department_number in COASTAL_DEPARTMENT_LIST:
         result = True
     return result
```
Comment (Quentame) [inline src/meteofrance/helpers.py:47]:
```suggestion
    return department_number in COASTAL_DEPARTMENT_LIST
```

File: src/meteofrance/helpers.py [hunk 2]
```diff
@@ -39,13 +40,24 @@ def get_phenomenon_name_from_indice(int_phenomenon: int, lang: str = "fr") -> st
...
 
+def is_valid_warning_department(department_number: str) -> bool:
+    """Identify if there is a weather alert bulletin for this department (Helper).
+
+    Weather alert buletins are available only for France métropole and Andorre.
+    """
+    result = False
+    if department_number in VALID_DEPARTMENT_LIST:
+        result = True
+    return result
```
Comment (Quentame) [inline src/meteofrance/helpers.py:58]:
```suggestion
    return department_number in VALID_DEPARTMENT_LIST
```

File: src/meteofrance/__init__.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
 """Météo-France API."""
-__version__ = "0.1.1-beta1"
+__version__ = "0.1.1"
```
Comment (Quentame) [inline src/meteofrance/__init__.py:2]: Version bump should be in a separate PR

Comment (oncleben31) [reply src/meteofrance/__init__.py:2]: OK will be done in another PR

**Comment by oncleben31** [comment]
And PR rebased on master.
